### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by tibetiroka**
This PR fixes some typos in the following categories:
 - Descriptions of outfits and ships
 - Conversations in missions or news
 - Help messages
 - README files
 - Code comments
 
 This PR does NOT fix typos in the following categories:
 - Names of categories, outfits, ships or people
 - Names of methods, fields or classes
 
 The second part is important to disclose, since there are some inconsistencies and typos in the project. Some notable examples:
  - Autopilot is sometimes treated as one word, sometimes as two (or with a hyphen), and this shows in method names as well
  - "Hand to Hand" is a category name, but is written with a hyphen. This is not corrected; however in the single case where this was written in an outfit's description it was changed in the text.
  - "self destruct" is also used in the engine, and is therefore not corrected. Same for text usage.

### CODE REVIEW

**Comment by Zitchas** [comment]
Overall these look like some good fixes. 

I haven't had the chance to look through all of them, but I'll note that at least one of the fixes for a missing `"` is actually intentional: If a continuous dialog or spoken element continues over multiple lines, then the trailing `"` is omitted until the actual end of the statement.

```
Sheri said "I think I need to write some dialog for an example.
"However, it needs to be broken up on multiple lines.
"I guess this makes the end of the statement."
```

https://style.mla.org/speech-paragraphs-quotation-marks/
https://www.grammar-monster.com/punctuation/quotation_marks_multiple_or_new_paragraphs.html

File: data/hai/hai culture conversations.txt [hunk 1]
```diff
@@ -197,7 +197,7 @@ mission "Gambling Judgment"
 			choice
 				`	"Some would say poker is more about skill than chance."`
 				`	"Surely if it's just a game though, there's no harm in it?"`
-			`	"Oh no, I'm afraid the first tenet ensures that it's really not favorable either way. The first is 'To have clear separation of fantasy and reality,' so a game cannot have real consequences or it is reality, not fantasy. So to gamble money on the game is to bring real consequences to a fantasy exercise. This is taboo for an adult.`
+			`	"Oh no, I'm afraid the first tenet ensures that it's really not favorable either way. The first is 'To have clear separation of fantasy and reality,' so a game cannot have real consequences or it is reality, not fantasy. So to gamble money on the game is to bring real consequences to a fantasy exercise. This is taboo for an adult."`
```
Comment (Amazinite) [inline data/hai/hai culture conversations.txt:200]: Not a typo.  http://baxtercommunications.nl/quotation-marks-multi-paragraph-rule/
```suggestion
			`	"Oh no, I'm afraid the first tenet ensures that it's really not favorable either way. The first is 'To have clear separation of fantasy and reality,' so a game cannot have real consequences or it is reality, not fantasy. So to gamble money on the game is to bring real consequences to a fantasy exercise. This is taboo for an adult.`
```

File: data/categories.txt [hunk 2]
```diff
@@ -21,7 +21,7 @@ category "ship"
 	"Drone"
 
 
-# Ships of these types are able to be carried by other ships.
+# Ships of these types can be carried by other ships.
```
Comment (Amazinite) [inline data/categories.txt:24]: Not a typo. How it was written is fine.
```suggestion
# Ships of these types are able to be carried by other ships.
```

File: data/help.txt [hunk 3]
```diff
@@ -14,7 +14,7 @@ help "bank"
 
 help "bank advanced"
 	`When entering a credit value at the bank, you can put in an exact number, or use the suffixes K, M, B, and T as shorthand for thousands, millions, billions, and trillions. For example, entering a value such as "100k" would count as 100,000 credits.`
-	`Entering a number that is higher than what you or the bank have available depending if you are paying or applying for a loan will default to the maximum available value. For example, entering "1m" to pay off a 100,000 credit loan will completely pay off the loan without giving extra, should you have available credits.`
+	`Entering a number that is higher than what you or the bank have available will default to the maximum available value. For example, entering "1m" to pay off a 100,000 credit loan will completely pay off the loan without giving extra, should you have available credits.`
```
Comment (Amazinite) [inline data/help.txt:17]: Not a typo.
```suggestion
	`Entering a number that is higher than what you or the bank have available depending if you are paying or applying for a loan will default to the maximum available value. For example, entering "1m" to pay off a 100,000 credit loan will completely pay off the loan without giving extra, should you have available credits.`
```

File: data/human/deep jobs.txt [hunk 4]
```diff
@@ -610,7 +610,7 @@ mission "Deep Security Package Reveal"
 				`	"Do all the secure packages belong to Deep Security?"`
 			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs, giving you a measured look.`
 			label blindacceptance
-			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion, and it seems like you qualified. You passed the background check at any rate.`
+			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion, and it seems like you qualified. You passed the background check at any rate."`
```
Comment (Amazinite) [inline data/human/deep jobs.txt:613]: Not a typo.
```suggestion
			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion, and it seems like you qualified. You passed the background check at any rate.`
```

File: data/intro missions.txt [hunk 5]
```diff
@@ -267,7 +267,7 @@ mission "Intro [2 Freighter]"
 			`	He pauses for a moment to think, then says, "Unless you want to do something really risky like hunting pirates, your best bet is to work with the Navy, or the Syndicate, or one of the local militias. Any of those would be a decent choice, although they've each got their own problems. If any of them offer you a mission, go ahead and take it; that's a good way to build connections and get to know the lay of the land."`
 				goto end
 			label career
-			`	"Well," he says, "Back when I was a bit older than you, I piloted a Firebird for a while, escorting merchant convoys up and down the Rim. I used to hope we'd get attacked; I'd disable the pirate ships and rob them blind. Turnabout is fair play, after all.`
+			`	"Well," he says, "Back when I was a bit older than you, I piloted a Firebird for a while, escorting merchant convoys up and down the Rim. I used to hope we'd get attacked; I'd disable the pirate ships and rob them blind. Turnabout is fair play, after all."`
```
Comment (Amazinite) [inline data/intro missions.txt:270]: Not a typo.
```suggestion
			`	"Well," he says, "Back when I was a bit older than you, I piloted a Firebird for a while, escorting merchant convoys up and down the Rim. I used to hope we'd get attacked; I'd disable the pirate ships and rob them blind. Turnabout is fair play, after all.`
```

File: data/intro missions.txt [hunk 6]
```diff
@@ -475,19 +475,19 @@ conversation "end of intro missions"
 		goto more
 	
 	label asteroids
-	`	"Well to start off, you'll want to stay right here in the Paradise Worlds. The asteroids aren't the best, but it's the safest place to mine while you're still in a small ship. But once you're in something that's able to take a hit, you'll want to explore for an uninhabited system with lots of asteroids. You'll encounter more pirates, but you'll likely find better asteroids.`
+	`	"Well to start off, you'll want to stay right here in the Paradise Worlds. The asteroids aren't the best, but it's the safest place to mine while you're still in a small ship. But once you're in something that's able to take a hit, you'll want to explore for an uninhabited system with lots of asteroids. You'll encounter more pirates, but you'll likely find better asteroids."`
```
Comment (Amazinite) [inline data/intro missions.txt:478]:
```suggestion
	`	"Well to start off, you'll want to stay right here in the Paradise Worlds. The asteroids aren't the best, but it's the safest place to mine while you're still in a small ship. But once you're in something that's able to take a hit, you'll want to explore for an uninhabited system with lots of asteroids. You'll encounter more pirates, but you'll likely find better asteroids.`
```

File: data/intro missions.txt [hunk 7]
```diff
@@ -475,19 +475,19 @@ conversation "end of intro missions"
...
 		goto more
 	
 	label aliens
-	`	"I've spoken with the Quarg a few times, but what captain hasn't? You'll find them down south by Tarazed. I haven't met any other aliens," he pauses for a moment, "as far as I'm aware, that is, but I'm sure they're out there if you go looking hard enough. I've explored all of Republic space in my many years, but there are far more systems out there that we don't even know how to reach.`
+	`	"I've spoken with the Quarg a few times, but what captain hasn't? You'll find them down south by Tarazed. I haven't met any other aliens," he pauses for a moment, "as far as I'm aware, that is, but I'm sure they're out there if you go looking hard enough. I've explored all of Republic space in my many years, but there are far more systems out there that we don't even know how to reach."`
```
Comment (Amazinite) [inline data/intro missions.txt:482]:
```suggestion
	`	"I've spoken with the Quarg a few times, but what captain hasn't? You'll find them down south by Tarazed. I haven't met any other aliens," he pauses for a moment, "as far as I'm aware, that is, but I'm sure they're out there if you go looking hard enough. I've explored all of Republic space in my many years, but there are far more systems out there that we don't even know how to reach.`
```

File: data/intro missions.txt [hunk 8]
```diff
@@ -475,19 +475,19 @@ conversation "end of intro missions"
...
 		goto more
 	
 	label trade
-	`	"One of the most reliable trade routes is to bring food from the Dirt Belt to Earth or the other urban worlds around it. There are billions of mouths to feed on those worlds, and the Dirt Belt sells food for cheap, so it's a great place to get started. Just remember though that if you use the same trade route for too long, you might end up causing the prices to stabilize. The Dirt Belt can only produce food and people can only eat it so fast, so change up your trade routes if you notice the return getting worse.`
+	`	"One of the most reliable trade routes is to bring food from the Dirt Belt to Earth or the other urban worlds around it. There are billions of mouths to feed on those worlds, and the Dirt Belt sells food for cheap, so it's a great place to get started. Just remember though that if you use the same trade route for too long, you might end up causing the prices to stabilize. The Dirt Belt can only produce food and people can only eat it so fast, so change up your trade routes if you notice the return getting worse."`
```
Comment (Amazinite) [inline data/intro missions.txt:486]:
```suggestion
	`	"One of the most reliable trade routes is to bring food from the Dirt Belt to Earth or the other urban worlds around it. There are billions of mouths to feed on those worlds, and the Dirt Belt sells food for cheap, so it's a great place to get started. Just remember though that if you use the same trade route for too long, you might end up causing the prices to stabilize. The Dirt Belt can only produce food and people can only eat it so fast, so change up your trade routes if you notice the return getting worse.`
```

File: data/intro missions.txt [hunk 9]
```diff
@@ -475,19 +475,19 @@ conversation "end of intro missions"
...
 		goto more
 	
 	label jobs
-	`	"Getting good jobs is more about having the space to accept the good jobs than it is finding them, but if you're looking for jobs that pay well for little effort, then you'll want to watch out around the holidays. There's always a big celebration on Earth every April 22nd, Earth Day, so pay attention to the job board in the months leading up to that. The people down in the Rim have some month-long celebration every August that you should look out for too.`
+	`	"Getting good jobs is more about having the space to accept the good jobs than it is finding them, but if you're looking for jobs that pay well for little effort, then you'll want to watch out around the holidays. There's always a big celebration on Earth every April 22nd, Earth Day, so pay attention to the job board in the months leading up to that. The people down in the Rim have some month-long celebration every August that you should look out for too."`
```
Comment (Amazinite) [inline data/intro missions.txt:490]:
```suggestion
	`	"Getting good jobs is more about having the space to accept the good jobs than it is finding them, but if you're looking for jobs that pay well for little effort, then you'll want to watch out around the holidays. There's always a big celebration on Earth every April 22nd, Earth Day, so pay attention to the job board in the months leading up to that. The people down in the Rim have some month-long celebration every August that you should look out for too.`
```

File: data/korath/korath missions.txt [hunk 10]
```diff
@@ -90,7 +90,7 @@ mission "Setar Fort Heating"
...
+			`	"As I am sure you are aware, <planet> is cold - uncomfortably so for most species. Based on our [research, information], the planet is experiencing an accelerated [descent, transition] into an ice age. The Kor Efreti have no effective means of stopping this, and their [pride, honor] means they will not accept our terraforming assistance unless they have no other options. They will, however, pay us for equipment to offset this [descent, transition]. Your [supplies, aid] would include warmer clothing, greenhouse building material, [insulation, shelter], and other technology for the Korath to use. The payment, while [unfortunately, sadly] low, is all the Korath were able to offer."`
```
Comment (Amazinite) [inline data/korath/korath missions.txt:93]:
```suggestion
			`	"As I am sure you are aware, <planet> is cold - uncomfortably so for most species. Based on our [research, information], the planet is experiencing an accelerated [descent, transition] into an ice age. The Kor Efreti have no effective means of stopping this, and their [pride, honor] means they will not accept our terraforming assistance unless they have no other options. They will, however, pay us for equipment to offset this [descent, transition]. Your [supplies, aid] would include warmer clothing, greenhouse building material, [insulation, shelter], and other technology for the Korath to use. The payment, while [unfortunately, sadly] low, is all the Korath were able to offer.`
```

File: data/quarg/quarg missions.txt [hunk 11]
```diff
@@ -201,7 +201,7 @@ mission "Lagrange Ring: Cargo"
 				`	"Are those other Captains also doing this? What do you need so much cargo for?"`
 				`	"I'm not looking for any more cargo jobs right now, sorry."`
 					decline
-			`	"Many have agreed to help us already, but we still need many more. Those that have agreed to help are headed all to different portions of human space, to safely bring us the materials we have ordered.`
+			`	"Many have agreed to help us already, but we still need many more. Those that have agreed to help are headed all to different portions of human space, to safely bring us the materials we have ordered."`
```
Comment (Amazinite) [inline data/quarg/quarg missions.txt:204]:
```suggestion
			`	"Many have agreed to help us already, but we still need many more. Those that have agreed to help are headed all to different portions of human space, to safely bring us the materials we have ordered.`
```

File: data/quarg/quarg missions.txt [hunk 12]
```diff
@@ -241,7 +241,7 @@ mission "Ask Quarg About Coalition Early"
 				`	(No.)`
 					decline
 			`	You stop the first Quarg that passes by you, and ask if they could tell their side of the story about the Coalition.`
-			`	"Ah, so you have encountered the foolish Coalition," it begins. "They once were three promising species, before they succumbed to the unwise desire for usurpation of the great deeds of others.`
+			`	"Ah, so you have encountered the foolish Coalition," it begins. "They once were three promising species, before they succumbed to the unwise desire for usurpation of the great deeds of others."`
```
Comment (Amazinite) [inline data/quarg/quarg missions.txt:244]:
```suggestion
			`	"Ah, so you have encountered the foolish Coalition," it begins. "They once were three promising species, before they succumbed to the unwise desire for usurpation of the great deeds of others.`
```

**Review by Amazinite — CHANGES_REQUESTED** [review]

File: data/help.txt (hunk 3 above)
Comment (tibetiroka) [reply data/help.txt:17]: I'm pretty sure that 'depending if' is not a correct from here. 'Depending on whether' or something like that would work better.

File: data/categories.txt (hunk 2 above)
Comment (tibetiroka) [reply data/categories.txt:24]: Literally every similar other sentence in that file uses 'can'. It is more concise and understandable (in my opinion, at least). Yes, technically the original version may be correct but it still feels to be out of place. Maybe we should get a third opinion on this.

File: data/help.txt (hunk 3 above)
Comment (EjoThims) [reply data/help.txt:17]: "Depending on if" would be the simplest typo fix.

Comment (Amazinite) [reply data/help.txt:17]: There's nothing wrong with "depending if." Any changes to this would be matters of preference.

Comment (tibetiroka) [reply data/help.txt:17]: >  "Depending on if" would be the simplest typo fix.

After prepositions, 'whether' is preferred over 'if':https://dictionary.cambridge.org/us/grammar/british-grammar/if-or-whether

> here's nothing wrong with "depending if." 

Depending needs a preposition. 'if' is **not** a preposition. 
https://lingohelp.me/preposition-after-adjective/depending-upon-or-depending-of/
https://www.espressoenglish.net/when-to-use-depend-vs-depending/

**Review by roadrunner56 — APPROVED** [review]

**Review by Amazinite — APPROVED** [review]

**Pull request merged**